package.json

```diff
@@ -55,46 +55,46 @@
   },
   "dependencies": {
     "babel-runtime": "6.11.6",
-    "element-resize-detector": "1.1.7",
+    "element-resize-detector": "1.1.8",
     "favico.js": "0.3.10",
     "jquery": "3.1.0",
     "jquery-sortable": "0.9.13",
     "konami": "1.3.3",
     "lodash": "4.15.0",
-    "moment": "2.14.1",
+    "moment": "2.15.0",
     "snudown-js": "1.4.0-2",
     "tinycolor2": "1.4.1"
   },
   "devDependencies": {
-    "autoprefixer": "6.4.0",
+    "autoprefixer": "6.4.1",
     "ava": "0.15.2",
-    "babel-cli": "6.11.4",
-    "babel-core": "6.13.2",
+    "babel-cli": "6.14.0",
+    "babel-core": "6.14.0",
     "babel-eslint": "6.1.2",
-    "babel-loader": "6.2.4",
-    "babel-plugin-lodash": "3.2.6",
-    "babel-plugin-transform-dead-code-elimination": "2.1.0",
-    "babel-plugin-transform-es2015-block-scoping": "6.10.1",
+    "babel-loader": "6.2.5",
+    "babel-plugin-lodash": "3.2.8",
+    "babel-plugin-transform-dead-code-elimination": "2.2.0",
+    "babel-plugin-transform-es2015-block-scoping": "6.15.0",
     "babel-plugin-transform-es2015-destructuring": "6.9.0",
-    "babel-plugin-transform-es2015-modules-commonjs": "6.11.5",
+    "babel-plugin-transform-es2015-modules-commonjs": "6.14.0",
     "babel-plugin-transform-es2015-parameters": "6.11.4",
     "babel-plugin-transform-node-env-inline": "6.8.0",
-    "babel-plugin-transform-runtime": "6.12.0",
-    "babel-preset-es2015": "6.13.2",
+    "babel-plugin-transform-runtime": "6.15.0",
+    "babel-preset-es2015": "6.14.0",
     "babel-preset-stage-0": "6.5.0",
     "chrome-extension-deploy": "2.0.3",
     "coveralls": "2.11.12",
-    "cross-env": "2.0.0",
-    "css-loader": "0.23.1",
+    "cross-env": "2.0.1",
+    "css-loader": "0.25.0",
     "deep-equal": "1.0.1",
     "eslint": "3.5.0",
-    "eslint-import-resolver-webpack": "0.5.1",
+    "eslint-import-resolver-webpack": "0.6.0",
     "eslint-plugin-ava": "3.0.0",
     "eslint-plugin-babel": "3.3.0",
     "eslint-plugin-consistent-return-legacy": "1.0.1",
     "eslint-plugin-dollar-sign": "0.0.5",
     "eslint-plugin-filenames": "1.1.0",
-    "eslint-plugin-import": "1.14.0",
+    "eslint-plugin-import": "1.15.0",
```

Comment:
One thing to do when updating eslint-* dependencies is to check if they have any new rules and add them if they make sense.

In this case, `eslint-plugin-import` has added [no-absolute-path](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-absolute-path.md), which looks like something we'd want to enforce.

So you can add `"import/no-absolute-path": 2,` to .eslintrc.json along with the other `import/` rules